test_unstructured/metrics/test_text_extraction.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from unstructured.metrics.text_extraction import calculate_edit_distance
+from unstructured.partition.auto import partition
+
+
+@pytest.mark.parametrize(
+    ("filename", "similarity", "percentage"),
```

Comment:
I see that similarity and percentage correspond to score and distance outputted by the function, but what do these terms mean, and why are they inverse of each other?

Reply:
since "score" (similarity) is the default return type from the function, why are two of these so low? is percentage the accuracy measure we actually want to use for marketing?

Reply:
Since we want to report the score of 100% being perfect, I did (1 - normalized distance) to get the score. And that's why they are the complement of each other's %. We can change the name accordingly, but I believe "score" is a good intuitive name of default result. I could change percentage -> distance and distance -> raw_distance if that makes more sense to you.

Reply:
Ok, score makes sense to me (could consider accuracy / correctness as names as well, but I like score too). 

> I could change percentage -> distance and distance -> raw_distance if that makes more sense to you.

Maybe we can do percentage -> edit_percentage (or percent_edit or some similar variation) and distance -> raw_distance ? I don't want them both to have the same name of "distance" since one is normalized and one is not.

Reply:
update: removing "percentage" and keeping only score and distance/raw_distance (name choice is up to you)